pay_ios/ios/Classes/PaymentHandler.swift

```diff
@@ -179,6 +188,11 @@ extension PaymentHandler: PKPaymentAuthorizationControllerDelegate {
   }
     
   func paymentAuthorizationController(_: PKPaymentAuthorizationController, didAuthorizePayment payment: PKPayment, handler completion: @escaping (PKPaymentAuthorizationResult) -> Void) {
+    // Store completion handler
```

Comment:
If the desrialization of `paymentResultData` in line 198 fails, this callback will remain populated and lead to state issues. Let's move it to where the former `completion` handler was placed (line 193).